### ISSUE TITLE
Deep Learning in Modern C++

### ISSUE THREAD

**Issue by chanelcolgate** [chanelcolgate/hands-on-iot#14]
- [ ] Introduction to Deep Learning Programming
- [ ] Coding Deep Learning with Modern C++
- [ ] Testing Deep Learning Code
- [ ] Implementing Covolutions
- [ ] Coding the Fully Connected Layer
- [ ] Learning by Minimizing Cost Functions
- [ ] Defining Activation Functions
- [ ] Using Pooling Layers
- [ ] Coding the Gradient Descent Algorithm
- [ ] Coding the Backpropagation Algorithm
- [ ] Underfitting Overfitting and Regularization
- [ ] Implementing Cross-validation, Mini Batching, and Model Performance Metrics
- [ ] Implementing Optimizers
- [ ] Introducing Computer Vision Models
- [ ] Developing an Image Classifier
- [ ] Leveraging Training Performance with Transfer Learning
- [ ] Developing an Object Localization System
